dotnet/csharp.bzl

```diff
@@ -48,50 +48,56 @@ def _get_libdirs(dlls, libdirs=[]):
   return [dep.dirname for dep in dlls] + libdirs
 
 def _make_csc_arglist(ctx, output, depinfo, extra_refs=[]):
+  args = ctx.actions.args()
+
   flag_start = ctx.attr._flag_start
```

Comment:
What's the point of having the `_flag_start` attribute if it's hidden (starts with "_") so the value is constant and the user needs to remember passing it to `_make_csc_flag` and it's always concatenated with the flag name? Wouldn't it be easier, more readable, and faster to use e.g. "-out" instead of "out" as flag names?

I won't block an LGTM on this issue but I think it'd be useful to fix.

Reply:
Hm, I think the original intent was a nod to supporting Windows: `csc` expects its flags to start with `/` rather than `-`. Ultimately this would become an attr of a toolchain.

Some quick googling led to: https://manpages.debian.org/wheezy/mono-mcs/mcs.1.en.html
> The Mono C# compiler accepts the same command line options that the Microsoft C# compiler does. Those options can start with a slash or a dash (/checked is the same as -checked).

I could confirm that on Linux/mono and then "inline" `/` into `_make_csc_flag` for simplicity. Does that sound good? As a follow-up PR?

Reply:
Sure. Thanks!